### DISCUSSION TITLE
还想弱弱的提一个问题，为什么有问题不在这公开交流，而是建立微信群呢？

### DISCUSSION THREAD

**Discussion by fschenkun22** [NervJS/taro#16899] [Q & A]
使用微信群qq群这些工具交流时候，很多信息都一闪而过，很难详细针对某一个问题进行反馈处理，为什么还要使用微信群呢

**Comment by TheKonka** [comment]
GitHub 公开交流一直是可以的，你说的详细针对某一个问题进行反馈处理不就是提issue吗？
群聊是提供更方便的渠道，毕竟大家使用微信比较多，交流也更频繁，有时候遇到啥小问题群里发下就能立马解决，如果啥大小问题都跑GitHub提issue 那效率会很低

**Reply by TurnOffNOD** [reply]
并不方便，因为群（包括微信群和qq群）只有时间线索，是一维的，而这里是时间和话题线索二维的。同时，这里是开放的，能被google收录，而微信（还有qq）是封闭的，不能被google收录。同时，微信群是不能看见在入群之前的消息的，也没有提问历史，是会造成一个简单问题反复被问的重复性工作。

群只是个吹水的地方，不适合做技术讨论。
